net/dnscrypt-proxy/Makefile

```diff
@@ -60,6 +73,7 @@ define Build/Configure
 	$(call Build/Configure/Default, \
 	--prefix=/usr \
 	--disable-ssp \
+	--disable-plugins \
```

Comment:
This option has been disabled by default in previous versions. By default it is enabled only in v1.7.0.

- This option slightly increases binary size and adds unnecessary dependencies (libltdl.so as I remember).
- This [plugins](https://dnscrypt.org/#plugins) can be replaced with dnsmasq.
- Technically plugins is just a native shared libraries. Shared libraries should be distributed using opkg. I do not see any useful third party plugins that can be added to the repository by developers. This feature can be enabled in the future if it will need plugin developers.